android/src/main/java/com/oney/WebRTCModule/StringUtils.java

```diff
@@ -74,6 +74,26 @@ private static void appendValue(StringBuilder builder, Object value) {
             }
 
             builder.append("]");
+        } else if (value instanceof Map) {
+            try {
+                Map<String, Object> mapValue = (Map) value;
+
+                boolean firstKey = true;
+                builder.append("{");
+
+                for (Map.Entry<String, Object> entry : mapValue.entrySet()) {
+                    if (firstKey) {
+                        firstKey = false;
+                    } else {
+                        builder.append(",");
+                    }
+                    builder.append("\"").append(entry.getKey()).append("\":");
+                    appendValue(builder, entry.getValue());
+                }
+                builder.append("}");
+            } catch (ClassCastException e) {
```

Comment:
What line can cause this? Looks like just `Map<String, Object> mapValue = (Map) value;`  Then let's keep it confined to that only line, and not append the value in that case, it'll likely be broken.

Reply:
Indeed that line might cause it. I have removed any handling of the value in case an exception is thrown, but confining the try catch to just the declaration of the variable would mean adding some extra logic that I think would look weird.